layouts/partials/announcement.html

```diff
@@ -15,7 +15,7 @@
     {{- if or (eq .endTime nil ) (gt ( time .endTime ) now ) -}}
       {{- if not $announcementShown -}}
       {{- $announcementShown = true -}}
-  <div lang="en" id="announcement" style="background-color: #3371e3; color: #fff; {{ .style | safeCSS }}">
+  <div lang="en" id="announcement" announcement-name={{ .name }} style="background-color: #3371e3; color: #fff; {{ .style | safeCSS }}">
```

Comment:
How about this?
(for consistency, seems more neat, ...)

```suggestion
  <div lang="en" id="announcement" announcement-name="{{ .name }}" style="background-color: #3371e3; color: #fff; {{ .style | safeCSS }}">
```

Reply:
Thanks for suggesting! I missed that out